src/photos/ducks/timeline-clusters/index.js

```diff
@@ -36,39 +38,122 @@ const TIMELINE_MUTATIONS = query => ({
     })
 })
 
-const getPhotosByMonth = photos => {
-  let sections = {}
-  photos.forEach(p => {
+const formatDate = date => {
+  return format(date, 'DD MMMM YYYY')
+}
+
+const sectionTitle = album => {
```

Comment:
Can we prefix this function with a verb to be consistent with the other ones (`getPhotosByClusters`, `handle…` etc)? Like `getSectionTitle` maybe?